src/connectors/horizonte.cl.js

```diff
@@ -9,7 +9,3 @@ Connector.trackSelector = 'h1[class="line__clamp_2"]';
 Connector.playButtonSelector = '.button_control .fa-play';
 
 Connector.trackArtSelector = '.np__thumbnail img';
-
-Connector.currentTimeSelector = '.np__thumbnail .current_time:first-child';
```

Comment:
Looks like this came in from the other PR  - can we remove these changes?

Reply:
Yes, sure! I had trouble to keep two PR separated.

Those changes should be part of [fb7d8dc](https://github.com/web-scrobbler/web-scrobbler/pull/3064/commits/fb7d8dc38697a437a0fff44865d42c35c08cea0a) only for horizonte.cl

Reply:
those changes were reverted, now it's only Klassikradio.